keylime-agent/src/config.rs

```diff
@@ -597,6 +597,12 @@ fn config_get_setting() -> Result<ConfigBuilder<DefaultState>, Error> {
     config_get_file_setting()
 }
 
+pub(crate) fn parse_list(list: &str) -> Vec<&str> {
+    list.split(&['"', ' ', ',', '[', ']', '\n', '\r'][..])
+        .filter(|x| !x.is_empty())
+        .collect()
+}
```

Comment:
TBH, that is a restriction I'm adding here, but I'm not sure is acceptable.

Maybe we need a more sophisticated parser that can deal with quoted strings properly so that we do not require the files names to be restricted.

I'll try to write such parser, and see if it gets too complicated.

Reply:
@ueno I implemented a simple parser for the lists, could you please check?